core-web/libs/dojo-theme/scss/backend/dot-admin/forms/_base.scss

```diff
@@ -92,8 +95,10 @@
 }
 
 .required2 {
-    color: $red;
-    font-weight: bold;
+    &::after {
+        color: $red;
+        content: $fa-var-asterisk;
+    }
```

Comment:
Can we unify this? so we don't have `required` and `required2` classes.

Reply:
there is some css difference in both classes  that's why I don't change it.

Reply:
ok sure

Reply:
updated